dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/BasicDeploymentForm.test.tsx

```diff
@@ -148,3 +148,36 @@ it("should hide an element if it depends on a param (object)", () => {
   const hiddenParam = wrapper.find("div").filterWhere(p => p.prop("hidden") === true);
   expect(hiddenParam).toExist();
 });
+
+it("should hide an element if it depends on multiple params (object)", () => {
+  const params = [
+    {
+      path: "foo",
+      type: "string",
+      hidden: {
+        conditions: [
+          {
+            value: "bar",
+            path: "enabled",
```

Comment:
there has been a change in the meaning. Before `value` meant the path to the element and `condition` the value it should have (yes, a bit weird). So for the object `value: mariadb.enabled, condition: true` it meant that `mariadb.enabled === true`.

You have added `path` but this `path` is actually the value, and the `value` is the path :). In your example, `path` should be `bar` (or if it were an inner object, it would be `bar/foo`) and the `value` should be `enabled`.

Hope I explained myself :)

Reply:
Yes, fixing it